### ISSUE TITLE
SDL : cannot create window

### ISSUE THREAD

**Issue by DeliciousBeanSauce** [tomassedovic/tcod-rs#316]
Hi,

I have followed the tutorial and even copy pasted the code to main.rs. Either way I end up with this error
`SDL : cannot create window
`
The error comes as soon as i use blit function. put_char works fine.

**Comment by tomassedovic** [comment]
Greetings @DeliciousBeanSauce!

It's really hard to tell how to fix this with the information provided.

Would you please try/answer (in as much detail as you can) the following?

* Which platform (linux, windows, macos) and what version are you testing this on?
* Did you install SDL2? How did you install it?
* What version of tcod-rs are you using?
* Have you tried the latest version from git? (add `tcod = { git = "https://github.com/tomassedovic/tcod-rs" }` under `[dependencies]` in your `Cargo.toml`)
* Can you clone the tcod-rs repository and run the examples? Like so:

```
$ git clone https://github.com/tomassedovic/tcod-rs
$ cd tcod-rs
$ cargo run --example keyboard
$ cargo run --example samples
```

Unfortunately, this repository is no longer maintained and I can't promise that myself or anyone else will be able to help you.

**Comment by DeliciousBeanSauce** [comment]
Thank You for the quick reply!
I am using Linux. I initially had issues with installing libsdl2-dev so I followed libsdl.org guide
```
hg clone https://hg.libsdl.org/SDL SDL
cd SDL
mkdir build
cd build
../configure
make
sudo make install
```
After that i could install libsdl2-dev

I am using 0.15.0 version of tcod-rs

I cannot run either of those examples. 
```
24 bits font.
key color : 0 0 0
24bits greyscale font. converting to 32bits
libtcod 1.6.3
SDL : cannot create window
```

**Comment by DeliciousBeanSauce** [comment]
Fixed, downloaded and compiled the latest 2.0.14 version and it works now. Renders the screen perfectly, as do the examples.
